File: STACK_QUEUE/기능개발.js
function solution(progresses, speeds) {
  const times = [];
  const answer = [];
  for (let i=0; i<progresses.length; i++){
      times.push(Math.ceil((100-progresses[i])/speeds[i]));
  }
  let target = times[0];
  let count = 0;
  for (let time of times){
      if (time>target){
          answer.push(count)
          count = 1;
          target = time;
      }else{
          count++;
      }
  }
  answer.push(count)

  return answer;
}

File: STACK_QUEUE/기능개발.py
import math


def solution(progresses, speeds):
    answer = []
    n = len(progresses)
    left_days = [math.ceil((100-progresses[i]) / speeds[i]) for i in range(n)]
    max_day = left_days[0]
    count = 0
    for day in left_days:
        if day > max_day:
            max_day = day
            answer.append(count)
            count = 1
        else:
            count += 1
    answer.append(count)
    return answer
